src/Microsoft.VisualStudio.SDK.Analyzers.Tests/MultiAnalyzerTests.cs

```diff
@@ -51,16 +46,20 @@ class Test {
 }
 ";
 
-        this.VerifyCSharpDiagnostic(test, vssdk: false);
+        await new CSharpTest
+        {
+            TestCode = test,
+            IncludeVisualStudioSdk = false,
+        }.RunAsync();
     }
 
-    protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => throw new NotImplementedException();
-
-    protected override ImmutableArray<DiagnosticAnalyzer> GetCSharpDiagnosticAnalyzers()
+    internal class CSharpTest : CSharpCodeFixTest<VSSDK001DeriveFromAsyncPackageAnalyzer, EmptyCodeFixProvider>
```

Comment:
>VSSDK001DeriveFromAsyncPackageAnalyzer [](start = 50, length = 38)

What is the significance that this one analyzer has compared to the others?

Reply:
None, I just needed to put some type there.